test/models/vote_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+describe Vote do
```

Comment:
You should also have a test on a validation that the combination user and ice_cream_id are unique.